packages/types/types/test.tsx

```diff
@@ -53,6 +61,14 @@ const mySpan = <Text.span regular /> // $ExpectType Element
 const myAbsolute = <Absolute bottom={1} top={1} left={1} right={1} /> // $ExpectType Element
 const myFlex = <Flex flexDirection={['row', null, 'column']} m={3} /> // $ExpectType Element
 const myHide = <Hide xs sm md lg xl xxl m={3} /> // $ExpectType Element
+const myCheckbox = <Checkbox id="test" size={1} onChange={() => {}}/> // $ExpectType Element
+const myFormField = <FormField /> // $ExpectType Element
+const myLabel = <Label /> // $ExpectType Element
+const myRadio = <Radio /> // $ExpectType Element
+const mySelect = <Select /> // $ExpectType Element
+const myTextArea = <TextArea id='test'/> // $ExpectType Element
+const myInput = <Input id='test'/> // $ExpectType Element
+const myInputGroup = <InputGroup /> // $ExpectType Element
```

Comment:
Can you add some test props to the new components? That seems to be a good way to validate prop types in addition to the expected return types